mikeio/dfs0.py

```diff
@@ -15,71 +16,80 @@
 from DHI.Generic.MikeZero.DFS.dfs0 import Dfs0Util
 
 from .dotnet import to_dotnet_array, to_dotnet_datetime, from_dotnet_datetime
-from .dutil import Dataset, find_item
+from .dutil import Dataset, find_item, get_valid_items_and_timesteps
 from .eum import TimeStep, EUMType, EUMUnit, ItemInfo
 from .helpers import safe_length
 
 
 class Dfs0:
-    def read(self, filename, item_numbers=None, item_names=None):
+    def __init__(self, filename=None):
```

Comment:
Shouldn't it be file_path and not filename across the code?

Reply:
filename is a path to a filename, I have checked some other python libraries and they use file, filename, filepath to mean the same thing, I think we will stick with filename.